doc/website/getting-started/list-of-examples.md

```diff
@@ -0,0 +1,16 @@
+# List of examples
+
+| Example                                         | Description                                                 | Level              |
+|:------------------------------------------------|:------------------------------------------------------------|:-------------------|
+|[icehello](./examples/icehello/)                 | How to send data to another process                         | :star:             |
+|[icedelivery](./examples/icedelivery/)           | How to send and receive data in a functional way            | :star:             |
```

Comment:
We do not send or receive data in a functional way, only the error handling is somehow functional when it is used in the `.and_then` `.or_else` fashion.

Reply:
And even this I would not mention here because it is functional programming which is meant and is misleading here.